src/components/dialogmoteikkeaktuell/DialogmoteikkeaktuellSkjema.tsx

```diff
@@ -0,0 +1,139 @@
+import React from "react";
+import { Link, Navigate } from "react-router-dom";
+import { moteoversiktRoutePath } from "@/routers/AppRouter";
+import { useDialogmotekandidat } from "@/data/dialogmotekandidat/dialogmotekandidatQueryHooks";
+import {
+  CreateIkkeAktuellDTO,
+  IkkeAktuellArsak,
+} from "@/data/dialogmotekandidat/types/dialogmoteikkeaktuellTypes";
+import { useValgtPersonident } from "@/hooks/useValgtBruker";
+import { useSettDialogmoteikkeaktuell } from "@/data/dialogmotekandidat/useSettDialogmoteikkeaktuell";
+import { SkjemaInnsendingFeil } from "@/components/SkjemaInnsendingFeil";
+import { SubmitHandler, useForm } from "react-hook-form";
+import {
+  Alert,
+  Box,
+  Button,
+  Radio,
+  RadioGroup,
+  Textarea,
+} from "@navikt/ds-react";
+
+export const texts = {
+  noBrev: "Det blir ikke sendt ut brev.",
```

Comment:
Er det digitalt brev vi sender ut i forbindelse med andre dialogmøte-vurderinger? Hva med `Det blir ikke sendt ut varsel eller brev til den sykmeldte`.

Reply:
Kanskje endre for unntak også da?

Reply:
Ja, eller jeg er bare usikker på om veileder skjønner at `brev == digitalt eller fysisk brev`, ikke bare fysisk brev. Men det sendes jo ikke varsel heller, så det er jo ikke feil å skrive begge 😬